spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -143,4 +143,34 @@
       end
     end
   end
+
+  context 'when using unsuported sync_mode' do
+    let(:params) do
```

Comment:
Similar code found in synchronize_user_filters_spec.rb:79, synchronize_user_filters_spec.rb:148 (mass = 52)